randovania/games/fusion/exporter/patch_data_factory.py

```diff
@@ -1,13 +1,249 @@
+            if hint_location == "AuxiliaryPower":
+                hints[hint_location] = " ".join(
+                    [text for _, text in charge_hint_mapping.items() if "has no need to be located" not in text]
+                )
+            elif hint_location == "RestrictedLabs":
+                hints[hint_location] = " ".join(
+                    [text for _, text in metroid_hint_mapping.items() if "has no need to be located" not in text]
+                )
+            else:
+                hints[hint_location] = " ".join(["Hint system still in development, we appreciate your patience."])
+
+        starting_items_list = item_names.additional_starting_equipment(
+            self.patches.configuration, self.patches.game, self.patches
+        )
+        starting_items_text = (
+            "HQ has provided you with the following starting items: " + ", ".join(starting_items_list) + ". "
+            if len(starting_items_list) > 0
+            else ""
+        )
+        metroid_location_text = "anywhere" if self.configuration.artifacts.prefer_anywhere else "at bosses"
+        for lang in hint_lang_list:
+            nav_text_json[lang] = {
+                "NavigationTerminals": hints,
+                "ShipText": {
+                    "InitialText": (
+                        f"{starting_items_text}Your objective is as follows: the [COLOR=3]SA-X[/COLOR] "
+                        f"has discovered and destroyed a top secret [COLOR=3]Metroid[/COLOR] breeding facility. "
+                        f"It released {self.patches.configuration.artifacts.placed_artifacts} "
+                        "infant Metroids into the station. "
+                        f"Initial scans indicate that they are hiding {metroid_location_text}. "
+                        f"Find and capture {self.patches.configuration.artifacts.required_artifacts} of them, "
+                        "to lure out the SA-X. "
+                        "Then initiate the station's self-destruct sequence. "
+                        "Uplink at [COLOR=2]Navigation Rooms[/COLOR] along the way. "
+                        "I can scan the station for useful equipment from there.[OBJECTIVE]Good. Move out."
+                    ),
+                    "ConfirmText": "Any Objections, Lady?",
+                },
+            }
+        return nav_text_json
+
+    def create_useless_pickup(self) -> PickupEntry:
+        """Used for any location with no PickupEntry assigned to it."""
+        return pickup_creator.create_nothing_pickup(
+            self.game.resource_database,
+            model_name="Empty",
+        )
+
+    def create_visual_nothing(self) -> PickupEntry:
+        """The model of this pickup replaces the model of all pickups when PickupModelDataSource is ETM"""
+        return pickup_creator.create_visual_nothing(self.game_enum(), "Empty")
+
     def create_game_specific_data(self) -> dict:
+        pickup_list = self.export_pickup_list()
+
+        mars_data = {
```

Comment:
Currently mars supports things that we dont. Writing them down here so that we have them collected. Should probably discuss elsewhere which options we want to provide / always set:
- option to disable demos on title screen
- Make door transitions instant instead of fast
- vanilla / "mars / zm missile limit
- unexplored map

Reply:
skip door transitions is in